packages/container/tests/container_test.js

```diff
@@ -535,3 +517,27 @@ QUnit.test('Lazy injection validations are cached', function() {
   container.lookup('apple:main');
   container.lookup('apple:main');
 });
+
+if (isEnabled('ember-container-inject-owner')) {
+  QUnit.test('A deprecated `container` property is appended to every instantiated object', function() {
+    const registry = new Registry();
```

Comment:
I find that super subtle, up-to-interpretation style stuff like this is NEVER followed. Make it concrete. Does the variable get rebound? No? Then it's `const`. `const` is default unless the variables needs to be rebound. How do you know it needs to be rebound, because you just tried to rebind it and your linter is yelling at you.